pkg/executor/types.go

```diff
@@ -28,6 +28,8 @@ type Executor interface {
 	// Details in: https://github.com/bacalhau-project/bacalhau/issues/2702
 
 	Run(ctx context.Context, Args *RunCommandRequest) (*models.RunCommandResult, error)
+	Start(ctx context.Context, request *RunCommandRequest) error
```

Comment:
Possibly the stupidest nit ever, but could `Start` be above `Run`?

Reply:
😂 you got it!